snapshot-labs/mana
# feat: add nonce manager

Closes: https://github.com/snapshot-labs/mana/issues/122
Depends on: https://github.com/snapshot-labs/sx.js/pull/247

### Test plan
- Run mana with sx.js linked
- Point to this mana instance on sx-ui.
- Create n proposals on some space (I just use simple space with Eth authenticator).
- Queue votes on all of those proposals at the same time (do not approve on Metamask).
- Once all queued up quickly confirm all signature requests.
- Everything should still work.
